### ISSUE TITLE
Buy button positioning

### ISSUE THREAD

**Issue by danielcrisp** [matthewcrisp/drumhut#8]
I don't think the two most important buttons are very obvious

![image](https://cloud.githubusercontent.com/assets/1104814/11531592/e5ffa7f2-98f5-11e5-9822-06564046f39c.png)

I think you should have them up by the price.

Remember the page might be viewed on mobile, so that pushes the buttons down even further:

![image](https://cloud.githubusercontent.com/assets/1104814/11531615/13ceb5d8-98f6-11e5-9952-4c292401106b.png)

**Comment by matthewcrisp** [comment]
Yeah, I'm still in the process of fiddling with the layout. Probably gonna get rid of the top banner too, to reduce scrolling.

Thank you again! The deploying worked :)
